### PR TITLE
Update bug-report

### PR DESCRIPTION
**Pull request by lornasong**
Changes:
 - Be explicit requesting generated files: terraform.tfvars, main.tf
 - Add section to request user-created task variable files
 - Improvements to section requisition config file

Preview of bug report:

Screen Shot 2020-10-13 at 4 02 56 PMScreen Shot 2020-10-13 at 3 54 41 PM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -28,27 +28,41 @@ Terraform v0.13.3
 ```
 
 ### Configuration File(s)
-```
-Copy config HCL here
-```
+_Reminder to redact any sensitive information that may be present in this file_
+
+<details>
```
Comment (eikenb) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]: Didn't know about this feature. Nice. :+1:

**Review by eikenb — APPROVED** [review]
Looks fine to me. 

It does seem like it might be getting kind of long and I'm concerned that might put some people off. But we should wait to see it in use before worrying to much about it.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (lornasong) [reply .github/ISSUE_TEMPLATE/bug_report.md:33]: @-findkim found this 🎉 !

**Comment by lornasong** [comment]
Thanks @eikenb! I agree that it's a bit long! One thing we can do to shorten it a bit is move the "Terraform Module" and "Task Variable File" sections into the "Additional Context" section as a sentence of two. But agree, we can wait and see before making the change.

**Pull request merged**